core/src/rpc.rs

```diff
@@ -35,6 +35,32 @@ use std::{
     time::{Duration, Instant},
 };
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct RpcResponseContext {
+    pub slot: u64,
```

Comment:
IMO let's just keep slot in here, we don't have a requirement for block/leader right now

Reply:
Agreed. I think we can eliminate this struct and just return a Slot in the response

Reply:
OK, I can do that. @mvines do you approve with de-`struct`-ing the `struct` and changing `context` to `slot`?

Reply:
🤔 Keeping the `struct` seems a little more future proof, maybe we'll want to add more context on day?   Does that seem ok @CriesofCarrots ?

Reply:
I can live with it. Just hoping to keep the code as simple as possible, especially since I'm going to have an ~~ugly~~ exciting rebase on this :)